manager/controlapi/common.go

```diff
@@ -94,8 +98,17 @@ func validateDriver(driver *api.Driver, pg plugingetter.PluginGetter, pluginType
 		return nil
 	}
 
-	p, err := pg.Get(driver.Name, pluginType, plugingetter.Lookup)
+	if pg != nil {
+		p, err = pg.Get(driver.Name, pluginType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(driver.Name, pluginType)
```

Comment:
I don't think it makes sense to add this alternative case, because as far as I know, if `pg` isn't provided, plugins aren't supported.

Also, won't leaving `p` nil cause a crash below when calling `p.IsV1`?